pkgs/applications/audio/melody/default.nix

```
@@ -0,0 +1,74 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, gtk3
+, pkgconfig
+, python3
+, desktop-file-utils
+, vala
+, wrapGAppsHook
+, appstream
+, clutter-gtk
+, cmake
+, gst_all_1
+, json-glib
+, libgee
+, libsoup
+, pantheon
+, sqlite
+, taglib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "melody";
+  version = "2.2.1";
+
+  src = fetchFromGitHub {
+    owner = "artemanufrij";
+    repo = "playmymusic";
+    rev = "bba5fe74134ad7533c41044d8ce41f1a8f69b3b0";
```

Use a tagged release if possible
```suggestion
    rev = "2.2.1";
```
If not using a tagged release, a reason should be given, and the version above should be changed to something like ``version = "unstable-2020-09-14"`` (where the date is the date of the commit).